plugin/trino-http-event-listener/pom.xml

```diff
@@ -91,7 +91,7 @@
         <dependency>
```

Comment:
We should be able to remove this dependency now. It shouldn't have been added in the first place.

Reply:
Actually maven complains that it's used by the test (which isn't the case and I don't understand why's that):

```
[INFO] --- maven-dependency-plugin:3.3.0:analyze-only (default) @ trino-http-event-listener ---
[WARN] Used undeclared dependencies found:
[WARN]    com.squareup.okio:okio-jvm:jar:3.0.0:test
```